duckduckgo.pot

```diff
@@ -25,6 +25,11 @@ msgstr ""
 msgid "%s blocked by safe search."
 msgstr ""
 
+#. We realise this is the incorrect way to handle plurals. Sorry, we are working on a fix.
+msgctxt "published dates for organic results"
+msgid "%s days ago"
+msgstr ""
```

Comment:
While this is not the correct way to handle plurals, there's a little plumbing I need to add in the back end to make plural handling work properly. Since this feature goes out in a couple of days, we're stuck with this hack for now.

Sorry all!